features/continue_as_new/signal/README.md

```diff
@@ -1,4 +1,4 @@
-# Continue as new
+# signal
```

Comment:
Maybe call this signals_block_completion.
I don't know if I'd put this test in continue-as-new, it's any workflow completion that's blocked by unprocessed signals coming in while the concluding task is processed by a worker.